lib/board.rb

```diff
@@ -1,23 +1,37 @@
 class Board
-  attr_reader :board, :moves_remaining
+  attr_reader :squares
 
-  def initialize(board)
-    @board = board
-    @moves_remaining = 9
+  def initialize(squares)
+    @squares = squares
+    @winning_combinations = [
```

Comment:
This could be a class variable.

Reply:
My suggestion: winning_lines would maybe be more idiomatic. Think of someone who was playing the game who knew nothing about programming. Which would make more sense? Which conveys more information?

Reply:
Thanks, Matt. Winning_lines is much clearer than winning_combination